ci-operator/step-registry/workers-rhel/azure-provision/workers-rhel-azure-provision-ref.yaml

```diff
@@ -0,0 +1,31 @@
+ref:
+  as: workers-rhel-azure-provision
+  from: upi-installer
+  commands: workers-rhel-azure-provision-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  env:
+  - name: RHEL_WORKER_COUNT
+    default: "2"
+    documentation: |-
+      The number of rhel worker to be created.
+  - name: RHEL_IMAGE
+    default: "8.5.2021121504"
```

Comment:
I think we can merge `RHEL_IMAGE_SKU` here too, like "RedHat:RHEL:8-lvm-gen2:8.5.2021121504"